hail/python/hailtop/humanizex.py

```diff
@@ -13,36 +13,39 @@
 
 def _fmt(s: Union[int, float], word: str) -> str:
     s = int(s)
+    if word in SHORTHAND_UNITS:
+        return f'{s}{word}'
     if s == 1:
         return f'{s} {word}'
     return f'{s} {word}s'
 
 
+SHORTHAND_UNITS = {'s', 'ms', 'μs'}
 _units = [
     (_MONTH, 'month'),
     (_WEEK, 'week'),
     (_DAY, 'day'),
     (_HOUR, 'hour'),
     (_MINUTE, 'minute'),
-    (_SECOND, 'second'),
-    (_MILLISECOND, 'millisecond'),
-    (_MICROSECOND, 'microsecond'),
+    (_SECOND, 's'),
+    (_MILLISECOND, 'ms'),
+    (_MICROSECOND, 'μs'),
```

Comment:
this won't display correctly on some browsers unless we set `<meta charset="utf-8">`. It also must appear in the first 1024 bytes. [details](https://www.w3.org/International/questions/qa-html-encoding-declarations). It appears to be set for the website, but we need to set it for web_common. I think that's it though?

Reply:
Interesting, didn't know you could be non-utf-8 these days.